### ISSUE TITLE
Rogue bug list

### ISSUE THREAD

**Issue by Francesq** [Wall-core/Everlook-Bugtracker#680]
Playing rogue for a while, I have help to find some bugs on another pserv so I have found some bugs here I will describe :

- Gouge bug / stun bug : under stun or gouge, the target usually do a fake half turn and the game is showing it turned but it is not (it seems it is a blizzlike bug but it is still a bug...)

- distract bug : distract is only supposed to work out of combat and stoping the target's run or breaking regen, kick a spell out of combat (ex : a mage casting blizzard when out of combat or a healer casting out of combat). On everlook distract is working in combat but it is just a waste of the skill cause it has no effect in combat. The skill is not supposed to work and an error message is supposed to be displayed like "target is in combat", preventing the use of this skill for nothing.

- out of combat bug : another very boring bug is why'd out of combat one : when you do the classical rotation like cheap shot, hemo, kidney, hemo, gouge (5.5 with talent points), at the end of the gouge you don't have time to restealth without using vanish. It is because the out of combat duration is too long. We are supposed to be out of combat after 5 sec so the 5.5 gouge (with 3 talent points) is suppose to make viable this cycle.

- stealth bug with normal combat bar : I did not find the origin but sometimes you have to press 2 times the key bind to stealth when using it in the first action bar (the one which is activated by default). The result is sometimes you wanna stealth before attacking a npc or a player but you don't stealth because the first press don't work...

- sap bug : sap delay is really too long. When using it on npc, it have time to hit before sap. It is maybe because sap removes stealth before performing but it should be the opposite : sap should be perform then only stealth should disappear (if no talent points)

- Earth binding totem is breaking stealth if not resisted and not breaking stealth if resisted. it should be the opposite.

**Comment by Gamemechanicwow** [comment]
No offense, but you may want to post some proof or at least research some of these "issues".

**Comment by Francesq** [comment]
What proofs do you need for it ?
Some things don't need to be proved : the time to be out of combat should be inferior to imp gouge duration (5.5 s) or playing pvp rogue is just a joke.
You can't restealth etc.
All really pvp players knows that.
I am just disappointed there was PTR before with the promise live will be clean but it is not and I don't want to spend my time finding things like that. I came first before I trust it but as much as I play I see it is not clean and there is pserv with fewer population with better script which is sad...

**Comment by Francesq** [comment]
> No offense, but you may want to post some proof or at least research some of these "issues".

Yes, we should always bring proof of issues unless we will just crash on to the wall (no offense  TheWall ;) but this should have been identify during ptr. I don't have time for it now (and who have time for it as a player ?). So I just help showing those issues for anybody who have time for it.

Immo one of the major one is the time to get out of combat which really impact rogue class cause pvp gameplay is based to get out of combat unless you just use really too fast your CC's and becomes vulnerable...

**Comment by Francesq** [comment]
> No offense, but you may want to post some proof or at least research some of these "issues".

For the out of combat bug.

I have seen this video posted on YouTube there is 15 years ago so no private server, only official ones.

https://youtu.be/8BXl_ZYiEqA

At 5:40

We can see before the end of the gouge (showed by NECB add on) stealth icon  becomes active which means he is out of combat in less than 5.5 sec, but he probably doesn't know it because he uses kidney shot instead of Cheap shot...

Sorry, quality is really bad...
